tests/test_m2m.py

```diff
@@ -44,30 +44,30 @@ def setUp(self):
                 </creator>
                 <meta />
             </metadata>
-        """
+        '''
         self.agent_tree = objectify.fromstring(agent_xml)
 
-        pub_xml = b"""<?xml version="1.0" encoding="UTF-8"?>
+        pub_xml = b'''<?xml version='1.0' encoding='UTF-8'?>
             <metadata>
                 <publisher>
                     <name />
                     <location />
                 </publisher>
                 <meta />
             </metadata>
-        """
+        '''
         self.pub_tree = objectify.fromstring(pub_xml)
 
     def test_metadata_record_setup(self):
-        # test that the creation of a record creates a MetadataRecord object.
+        """Tests that the creation of a record creates a MetadataRecord object."""
```

Comment:
Pep 257 recommends a [command for the docstring](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) instead of a description, i.e.  `Test that` not `Tests that`.